### PR TITLE
SuiteP Theme

### PR DESCRIPTION
**Pull request by daniel-samson**
#SuiteP changes/Fixes

## Description
Changed list view, detail view, form elements, main navigation.
Moved some hardcoded stuff out into a tpl.

## Motivation and Context
Theme Update

## How To Test This
Look at the list view on all devices.
Look at the detail view.

## Types of changes
css, js, tpl, php.

### CODE REVIEW

File: include/javascript/jquery/jquery.sugarMenu.js [hunk 1]
```diff
@@ -167,14 +167,14 @@
...
 					}
-						if(menuNode.hasClass("fancymenu")){
-							dropDownHandle.addClass("ab");
-							dropDownHandle.tipTip({maxWidth: "auto",
-							   edgeOffset: 10,
-		                       content: "More Actions",
-		                       defaultPosition: "top"});
-
-						}
+						//if(menuNode.hasClass("fancymenu")){
```
Comment (mattlorimer) [inline include/javascript/jquery/jquery.sugarMenu.js]: remove don't comment out

File: include/Smarty/Smarty.class.php [hunk 2]
```diff
@@ -1117,8 +1117,22 @@ function display($resource_name, $cache_id = null, $compile_id = null)
      */
     function fetch($resource_name, $cache_id = null, $compile_id = null, $display = false)
     {
+        global $current_user, $sugar_config;
```
Comment (mattlorimer) [inline include/Smarty/Smarty.class.php]: override in include/Sugar_Smarty.php bit directly in third party smart.class

File: themes/SuiteP/css/yui.css [hunk 3]
```diff
@@ -1,11 +1,7 @@
-/**
- *
+/*********************************************************************************
```
Comment (mattlorimer) [inline themes/SuiteP/css/yui.css]: Licence regresion

File: themes/SuiteP/include/DetailView/DetailView.tpl [hunk 4]
```diff
@@ -0,0 +1,218 @@
+{*
+/*********************************************************************************
```
Comment (mattlorimer) [inline themes/SuiteP/include/DetailView/DetailView.tpl]: Old Licence Header

File: themes/SuiteP/include/DetailView/tab_panel_content.tpl [hunk 5]
```diff
@@ -0,0 +1,54 @@
+            {*{{if !is_array($panel)}}*}
```
Comment (mattlorimer) [inline themes/SuiteP/include/DetailView/tab_panel_content.tpl]: remove don't comment out

File: themes/SuiteP/js/style.js [hunk 6]
```diff
@@ -268,33 +268,48 @@ $(function() {
...
-            $('.search_form').toggle();
-        });
-        $('#userlinks_togglemobilesearch').click(function() {
-            $('#searchmobile').toggle('slide', {direction: 'left'}, '350');
-        });
-
-        clearInterval(checkContents);
-    }
-},1);
+//var checkContents = setInterval(function(){
```
Comment (mattlorimer) [inline themes/SuiteP/js/style.js]: remove don't comment out

**Comment by mattlorimer** [comment]
invalid change to themes/default/images/company_logo.png, verify change to themes/default/images/id-ff-clear.png and themes/default/images/id-ff-select.png?

**Comment by samus-aran** [comment]
@daniel-samson @paulgodley Be careful of editing in github's web interface - there is an issue with clahub that doesn't recognised the logged in user.  Please commit via your local instance up.

Going forward - we will need to break down large project PRs into more manageable sections.  After committing a reasonable size PR create a new branch to continue working.  PRs will be reviewed and commented on whilst you create new PRs with your progressed branches.  When a PR has been reviewed and updated I would suggest you cherry pick your PR specific commits (surely there wouldn't be many) into your progressed branches (where relevant to ensure you are kept up to date.

File: themes/SuiteP/include/DetailView/header.tpl [hunk 7]
```diff
@@ -38,6 +38,8 @@
  ********************************************************************************/
 
 *}
+
```
Comment (samus-aran) [inline themes/SuiteP/include/DetailView/header.tpl]: Watch your extras spaces

File: themes/SuiteP/css/style.css [hunk 8]
```diff
@@ -1876,10 +1876,15 @@ p.error {
     margin: 0;
 }
 
+
```
Comment (samus-aran) [inline themes/SuiteP/css/style.css]: Control the extra spaces

File: themes/SuiteP/include/DetailView/tab_panel_content.tpl [hunk 9]
```diff
@@ -62,23 +63,26 @@
...
-				{{if !empty($colData.field.name)}}
-				{/if}
-				{{/if}}
-                    </div>
-                {{/if}}
-
+				    {{$colData.field.suffix}}
+				    {{if !empty($colData.field.name)}}
+            {/if}
+                    {{else}}
```
Comment (samus-aran) [inline themes/SuiteP/include/DetailView/tab_panel_content.tpl]: Are these else needed... seems like if else do nothing?

File: themes/SuiteP/include/DetailView/tab_panel_content.tpl [hunk 10]
```diff
@@ -62,23 +63,26 @@
...
+                    {{/if}}
+                </div>
+                {{else}}
```
Comment (samus-aran) [inline themes/SuiteP/include/DetailView/tab_panel_content.tpl]: Same here - else do nothing?

**Comment by daniel-samson** [comment]
I have amended my pull request to consider your comments.

File: include/Sugar_Smarty.php [hunk 11]
```diff
@@ -105,4 +105,223 @@ function _unlink($resource, $exp_time = null)
...
+		$theme_directory = $current_theme->dirName;
+		if(strpos($resource_name, "themes".DIRECTORY_SEPARATOR.$theme_directory) === false) {
+			$test_path = SUGAR_PATH.DIRECTORY_SEPARATOR."themes".DIRECTORY_SEPARATOR.$theme_directory.DIRECTORY_SEPARATOR.$resource_name;
+			if(file_exists($test_path)) {
+				$resource_name = "themes".DIRECTORY_SEPARATOR.$theme_directory.DIRECTORY_SEPARATOR.$resource_name;
+			}
+		}
+		///
+
+		$_smarty_old_error_level = $this->debugging ? error_reporting() : error_reporting(isset($this->error_reporting)
```
Comment (mattlorimer) [inline include/Sugar_Smarty.php]: why do you need to copy the full function? can you not call the the parent function

Comment (daniel-samson) [inline include/Sugar_Smarty.php]: When I use the parent function, the view doesn't show. Perhaps this is related to the static $_cache_info.

**Comment by daniel-samson** [comment]
I change the fetch to use the parent::fetch(...) call.

**Pull request closed without merging**